python/src/space/core/storage.py

```diff
@@ -510,6 +580,9 @@ def __enter__(self) -> Transaction:
     # mutations.
     self._storage.reload()
     self._snapshot_id = self._storage.metadata.current_snapshot_id
+    if self._branch != _MAIN_BRANCH:
```

Comment:
It is used twice, can we make it a method `current_snapshot_id` in `storage`?

self._snapshot_id = self._storage.metadata.current_snapshot_id
if self._branch != _MAIN_BRANCH:
  self._snapshot_id = self._storage.lookup_reference(self._branch).snapshot_id

Reply:
Done